tensorflow_datasets/image/fruits360_test.py

```diff
@@ -0,0 +1,43 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for fruits360 dataset module."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow_datasets import testing
+from tensorflow_datasets.image import fruits360
+
+num_classes = len(fruits360._CLASS_NAMES)
+
+
+class FruitsTest(testing.DatasetBuilderTestCase):
+    DATASET_CLASS = fruits360.Fruits360
+
+    SPLITS = {
+        "train": 2 * num_classes,
+        "test": 2 * num_classes,
+    }
+
+    DL_EXTRACT_RESULT = {
+        'train': 'Fruit-Images-Dataset-%s/Training' % fruits360._COMMIT_SHA,
+        'test': 'Fruit-Images-Dataset-%s/Test' % fruits360._COMMIT_SHA
+    }
+
```

Comment:
Can u please tell me what is in the `.h5` file or how is the dataset structured ?

Reply:
The `.h5` file is generated by the `testing/fruits360.py` file. It contains `images` which are `uint8`s of the shape `(100, 100, 3)`, and `labels` which are `int64`s.

The dataset is downloaded from a Github archive [here](https://github.com/Horea94/Fruit-Images-Dataset/). There are two folders of interest: the `Training` and `Test` folders. Each contains sub-folders for every label which in turn contain the images as separate files.

Reply:
Actually you should create the fake_samples exactly like in the original dataset. 
So, your fake_samples should have something like - 
```
fruits-360-> |
             |-> Test 
             |-> Training
             |-> test-multiple_fruits 
```
Its, Ok to use less number of fruits, than in the original dataset.

Reply:
Ah, I see. Do I need to include actual fruit images or just fake images of the correct size?

Reply:
@ChanchalKumarMaji I just pushed a new fake_samples directory/generator to mirror the structure of the dataset. I did not include the `test-multiple_fruits` set because that would probably constitute its own `DatasetBuilder` since it has a different structure.

Reply:
Yes, make sure your images are all fake, not taken from the original dataset. Thanks.